ANGEL_URREGO_1011081750/Tarea-6-Clase-8/punto2.js

```diff
@@ -0,0 +1,19 @@
+//PUNTO #2
+const estudiantes = [
+    { nombre: "Camilo", edad: 33, promedio: 10 },
+    { nombre: "Juan", edad: 30, promedio: 6 },
+    { nombre: "Andres", edad: 29, promedio: 8.7 },
+];
+
+// EJERCICIO#1 Impreme el nombre del segundo estudiante
+const segundoEstudiante = estudiantes.find((element, index) => index === 1);
+console.log(segundoEstudiante.nombre); 
+
+// EJERCICIO#2 Actualiza la informacion del primer estudiante
+const updateAge = estudiantes.filter(estudiante => {
```

Comment:
aqui no es miy conveniente usar filter ya que filter devuelve un array nuevo que pase una condicion, aqui seria mejor usar el forEach porque no necesitamos un retorno si no actualizar un estudiante en especifico y ya...es mas no necesitamos ni for ni nada, solo con la posicion [0] podrias hacerlo